nodemon.json

```diff
@@ -0,0 +1,6 @@
+{
+    "watch": ["src"],
+    "ext": "ts",
+    "ignore": ["src/**/*.spec.ts"],
+    "exec": "ts-node ./src/index.ts"
```

Comment:
Is `ts-node` needed, I would prefer to use the production code so there are no surprise with `npm run start:local`. What do you think?

Reply:
I'll double check to see if ts-node is needed.

Reply:
Thank you 👍